src/index.js

```diff
@@ -43,6 +41,10 @@ subscriptionClient.onConnected(() => (connectionAttempts = 0))
 
 // Check for connection errors and if reaches max attempts send error log to Sentry
 subscriptionClient.onError(err => {
+  if (env('MOCK_DATA')) {
```

Comment:
Could we totally ignore subscriptions if we're in the mocked env?

Reply:
I'm not totally sure if we could ignore them completely because we are using a custom subscription exchange which fetches the data from the `mock` folder when we are in the mocked env. See https://github.com/aragon/court-dashboard/blob/b560acc7298656904a5853953eb3aa471dfe26af/src/mock/exchanges/index.js#L13 
If we were to discard using subscriptions then i think we would have to change all the `subscription-hooks` logic.

Reply:
Gotcha, that makes sense!